src/pages/Login/Login.js

```diff
@@ -1,8 +1,136 @@
 import React, { Component } from 'react';
+// import { link } from 'react-router-dom';
+import './Login.scss';
 
 class Login extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      id: '',
+      pw: '',
+      btn: 'btnLogin',
+    };
+  }
+  onchangeId = e => {
+    this.setState({
+      id: e.target.value,
+    });
+  };
+  onchangePw = e => {
+    this.setState({
+      pw: e.target.value,
+    });
+  };
```

Comment:
```suggestion
  handleInputs = e => {
    const { name, value } = e.target;
    this.setState({
      [name]: value,
    });
  };
```
const {name, value} = e.target;
이 부분은 '비구조화할당' 이란 키워드로 검색해보세요!

또한 해당 함수가 바뀜에 따라 하위에 있는 input에 name 값을 부여하고, onchange시 실행되는 함수명도 변경해주세요 :)